controllers/upgrade.go

```diff
@@ -111,13 +111,21 @@ func (r *RollingUpgradeReconciler) ReplaceNodeBatch(rollingUpgrade *v1alpha1.Rol
 			// Add in-progress tag
 			if err := r.Auth.TagEC2instance(instanceID, instanceStateTagKey, inProgressTagValue); err != nil {
 				r.Error(err, "failed to set instance tag", "name", rollingUpgrade.NamespacedName(), "instance", instanceID)
+				return false, err
 			}
 
 			// Standby
+			if err := r.SetInstanceStandBy(target, rollingUpgrade); err != nil {
+				r.Error(err, "couldn't set the instance to stand-by", "name", rollingUpgrade.NamespacedName(), "instance", instanceID)
+				return false, err
+			}
 
 			// Wait for desired nodes
+			if !r.DesiredNodesReady(rollingUpgrade) {
```

Comment:
Should we put a log line here instead of the internal logging?

Reply:
You mean, instead of error logging?